libpod/image/image_test.go

```diff
@@ -192,6 +193,51 @@ func TestImage_MatchRepoTag(t *testing.T) {
 	cleanup(workdir, ir)
 }
 
+// TestImage_RepoDigests tests RepoDigest generation.
+func TestImage_RepoDigests(t *testing.T) {
+	dgst, err := digest.Parse("sha256:7173b809ca12ec5dee4506cd86be934c4596dd234ee82c0662eac04a8c2c71dc")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, test := range []struct {
+		name     string
+		names    []string
+		expected []string
+	}{
+		{
+			name:     "empty",
+			names:    []string{},
+			expected: nil,
+		},
+		{
+			name:     "tagged",
+			names:    []string{"docker.io/library/busybox:latest"},
+			expected: []string{"docker.io/library/busybox@sha256:7173b809ca12ec5dee4506cd86be934c4596dd234ee82c0662eac04a8c2c71dc"},
```

Comment:
How often does Busybox update (such that this would no longer pull the same image)?

Can we throw a comment on here that this needs updating when busybox bumps its image version?

Reply:
> How often does Busybox update...

This digest is hard-coded in the test itself; it is completely decoupled from any real-world BusyBox image.  So no need to bump anything.

Reply:
Oh, sorry, I meant the line above - `busybox:latest`

Reply:
> Oh, sorry, I meant the line above - `busybox:latest`

Still just a string I'm hard-coding for these unit tests with no relation to real-world images.  The new unit tests should just be exercising vendored string-manipulation code; they aren't hitting the network or anything.